custom_dc/troubleshooting.md

```diff
@@ -45,10 +45,10 @@ This indicates that you have not specified API keys in the environment file. Fol
 
 ### "SQL schema check failed"
 
-This error indicates that there is a problem with the database schema. Check for the following additional errors:
+This error indicates that there is a problem with the database schema. Check for the following additional error:
+
+- "The following columns are missing..." -- This indicates that there is has been an update to the schema in the latest Docker build. To remedy this, rerun the data management Docker container and then restart the services container.
```

Comment:
nit: Perhaps we can drop the docker build part and just say "This indicates that there is has been an update to the database schema. To remedy this..."

Reply:
OK